src/components/Registration.jsx

```diff
@@ -1,65 +1,85 @@
-import { useState } from "react";
+import { useState } from 'react';
+import headerLogo from '../logos/mainLogo.svg';
+import bodyLogo from '../logos/Buildings.svg';
 
 const Registration = ({ isRegistr, setRegistrated }) => {
-  const [login, setLogin] = useState("");
-  const [password, setPassword] = useState("");
-  const [passwordRepeat, setPasswordRepeat] = useState("");
+  const [login, setLogin] = useState('');
+  const [password, setPassword] = useState('');
+  const [passwordRepeat, setPasswordRepeat] = useState('');
   const passwordVerify = (login, password, passwordRepeat) => {
-    const registr = /^(?=.*[0-9])(?=.*[a-zA-Z])[a-zA-Z0-9]{6,16}$/;
+    const rule = /^(?=.*[0-9])(?=.*[a-zA-Z])[a-zA-Z0-9]{6,16}$/;
     if (
-      registr.test(password) &&
-      login.length > 5 &&
+     login.length > 5 &&
+      rule.test(password)  &&
       passwordRepeat === password
     ) {
-      console.log("registrated");
-    } else {
-      alert(`Длина логина должна быть не менее 6 символов! Длина пароля должна быть не менее 6 символов, обязательно содежать латинские буквы 
-      и содержать хотя-бы одну цифру!`);
-    }
-    return (
-      <div className="registrForm">
-        <h1 className="logHeader">Регистрация</h1>
-        <div className="logInputs">
-          <p>Логин</p>
+      console.log('registrate sucessful');
+    } else if(!(login.length > 5)){
+      alert(`Длина логина должна быть не менее 6 символов!`);
+    } else if(!(rule.test(password))){
+      alert(`Длина пароля должна быть не менее 6 символов, обязательно содежать латинские буквы и содержать хотя-бы одну цифру!`);
+    } else if (!(passwordRepeat === password)) {
+      alert(`Пароль или его повтор неверен! Пожалуйста, проверьте свои введенные данные!`);
+    };
+  };
+
+  return (
+    <div className='registrationForm'>
+      <div className='registrationHeader'>
+        <div className='headerLogo'>
+          <img className='logoHeader' src={headerLogo} />
+        </div>
+        <div className='headerText'>
+          <p>Зарегистрироваться в системе</p>
+        </div>
+      </div>
+      <div className='registrationBody'>
+      <div className = 'bodyLogo'>
+        <img className = 'logoBody' src={bodyLogo} />
+      </div>
+      <div className='registrationField'>
+        <h1 className='loginHeader'>Регистрация</h1>
+        <div className='loginInputs'>
+          <div className='fieldContent'><p>Логин:</p></div>
           <input
-            classname="registrLogInput"
-            type="text"
-            placeholder="Логин"
+            className='loginInput'
+            type='text'
+            placeholder='Логин'
             value={login}
             onChange={(e) => setLogin(e.target.value)}
```

Comment:
take it out into one common universal function and use it in the password, and do not create new ones